PWGCF/TableProducer/filterCorrelations.cxx

```diff
@@ -98,6 +98,12 @@ struct FilterCF {
       return collision.alias_bit(kINT7) && collision.sel7();
     } else if (cfgTrigger == 8) {
       return collision.sel8();
+    } else if (cfgTrigger == 9) { // relevant only for Pb-Pb
+      return collision.sel8() && collision.selection_bit(aod::evsel::kNoSameBunchPileup) && collision.selection_bit(aod::evsel::kIsGoodZvtxFT0vsPV);
+    } else if (cfgTrigger == 10) { // TVX trigger only (sel8 selection before April, 2024)
+      return collision.selection_bit(aod::evsel::kIsTriggerTVX);
```

Comment:
isn't this missing `collision.sel8() &&`?

Reply:
No, I think we should not have `collision.sel8()` for this option as in the new version we have additional kNoTimeFrameBorder and kNoITSROFrameBorder cuts which were not present before. 
So, we need only the first part of 
`bool sel8 = bc.selection_bit(kIsTriggerTVX) && bc.selection_bit(kNoTimeFrameBorder) && bc.selection_bit(kNoITSROFrameBorder);`